### PR TITLE
[Kotlin Migration] DiffEngine

### PR DESCRIPTION
**Pull request by Akshay0701**

### CODE REVIEW

**Comment by Akshay0701** [comment]
solving errors....

File: AnkiDroid/src/main/java/com/ichi2/utils/DiffEngine.kt [hunk 1]
```diff
@@ -39,48 +34,44 @@ public class DiffEngine {
...
-        return "<span class=\"typeBad\">" + escapeHtml(in) + "</span>";
+        return arrayOf(prettyTyped.toString(), prettyCorrect.toString())
     }
 
-    @CheckResult
-    public static String wrapGood(String in) {
-        return "<span class=\"typeGood\">" + escapeHtml(in) + "</span>";
-    }
+    companion object {
+        private fun wrapBad(`in`: String): String {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/utils/DiffEngine.kt:54]: Cleanup: `in`

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
Besides the errors, one nit

**Review by david-allison-1 — APPROVED** [review]
LGTM

**Review by mikehardy — APPROVED** [review]

**Pull request merged**